app/static/js/courseManagement.js

```diff
@@ -3,13 +3,25 @@ function changeTerm() {
     $('form').submit();
 };
 
+function courseAction(action){
+  courseID = action.id;
+  if (action.value == "Review"){
+    reviewCourses(courseID)
+  }
+  else if (action.value == "View"){
+    location = '/serviceLearning/viewProposal/' + courseID
+  }
+  else if (action.value == "Edit"){
+    location = '/serviceLearning/editProposal/' + courseID
+  }
+}
+
 function formSubmit(el) {
   $("#termSelector").attr('action', '/manageServiceLearning/' + el);
   $("#termSelector").submit()
 };
 
-function reviewCourses(el) {
-  let courseID = $(el).data('id');
+function reviewCourses(action) {
```

Comment:
You call this variable `action` and don't use it. You should rename it to `courseID` to match how you use it. The only reason it works now is because of the global variable set in `courseAction`.